digid_eherkenning/oidc/models/base.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.utils.functional import classproperty
+from django.utils.module_loading import import_string
+from django.utils.translation import gettext_lazy as _
+
+from mozilla_django_oidc_db.models import OpenIDConnectConfigBase
+
+
+def get_default_scopes_bsn():
+    """
+    Returns the default scopes to request for OpenID Connect logins
+    """
+    return ["openid", "bsn"]
+
+
+def get_default_scopes_kvk():
+    """
+    Returns the default scopes to request for OpenID Connect logins
+    """
+    return ["openid", "kvk"]
+
+
+class OpenIDConnectBaseConfig(OpenIDConnectConfigBase):
+    """
+    Base configuration for DigiD/eHerkenning authentication via OpenID Connect.
+    """
+
+    class Meta:
+        abstract = True
+
+    @classproperty
+    def oidcdb_check_idp_availability(cls) -> bool:
+        return True
+
+    def get_callback_view(self):
```

Comment:
Is this a preferable way to import it or would we have a problem with just an import in the method itself?

Reply:
the project can define the setting `DIGID_EHERKENNING_OIDC_CALLBACK_VIEW`, and in the settings you cannot import the view yet because it will likely import some models, which requires settings to be configured, so to make this lazy, we use a string with a thing to import (like django middleware/auth backends/...).

For simplicity's sake, there's a single mechanism for the imports here - providing a dotted path string to import.